src/v2/Apps/Auction/Components/AuctionDetails/__tests__/AuctionDetails.jest.tsx

```diff
@@ -57,10 +57,20 @@ describe("AuctionDetails", () => {
   it("shows formatted start time", () => {
     const wrapper = getWrapper({
       Sale: () => ({
-        formattedStartDateTime: "Starting in 20 minutes",
+        auctionsDetailFormattedStartDateTime: "Mar 22, 2022 • 9:22pm GMT",
       }),
     })
-    expect(wrapper.text()).toContain("Starting in 20 minutes")
+    expect(wrapper.text()).toContain("Mar 22, 2022 • 9:22pm GMT")
+  })
+
+  it("show the interval label", () => {
+    const wrapper = getWrapper({
+      Sale: () => ({
+        auctionsDetailCascadingIntervalLabel:
+          "Lots close in 1 minute intervals",
+      }),
+    })
+    expect(wrapper.text()).toContain("Lots close in 1 minute intervals")
```

Comment:
#minor but just realized this should read `1-minute intervals` with the `-`. no need to fix now in MP but maybe add a quick ticket so we don't forget?

Reply:
sounds good will do